pyproject.toml

```diff
@@ -89,7 +89,7 @@ dependencies = [
     "sqlalchemy>=1.4, <2",
     "sqlalchemy-utils>=0.38.3, <0.39",
     "sqlglot>=23.0.2,<24",
-    "sqlparse>=0.4.4, <0.5",
+    "sqlparse>=0.4.4, <0.6",
```

Comment:
Unless we know there is a problem with `0.6`, I would not put an upper bound. And if there is a know issue, we should put a clear comment as to what the known blocker is right above.

Maybe that's a topic for the devX group though, but when we unleash `supersetbot bump-python` it will only generate PRs that bumps `requirements/*.txt` within the ranges specified in `pyproject.toml`, so generally we'll want no upper bound to see the PR come through and whether CI passes.